art/api/tests/tests_models.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+
+from ..models import AssetType
+
+class AssetTypeModelTest(TestCase):
+    """Tests for the Asset Types Model"""
+    def setUp(self):
+        test_asset = AssetType(type="TestAsset")
+        test_asset.save()
+
+    def test_add_new_asset_type(self):
+        """Test add new asset type in model"""
+        new_assettype = AssetType(type="Macbook")
+        new_assettype.save()
+        all_assettypes = AssetType.objects.all()
+        self.assertEqual(len(all_assettypes), 2)
+
+    def test_edit_asset_type(self):
+        """Test edit an asset type in model"""
+        get_asset = AssetType.objects.get(type="TestAsset")
+        get_asset.type = "New Name"
+        get_asset.save()
+        self.assertEqual(len(AssetType.objects.all()), 1)
+        get_asset = AssetType.objects.get(type="New Name")
+        self.assertEqual(get_asset.type, "New Name")
+
+    def test_delete_asset_type(self):
+        """Test delete an asset type in model"""
+        get_asset = AssetType.objects.get(type="TestAsset")
+        get_asset.delete()
+        self.assertEqual(len(AssetType.objects.all()), 0)
```

Comment:
same here.. compare count before and after deletion.

Reply:
Will do